README.md

```diff
@@ -1,2 +1,45 @@
-# toDo
-A web based application to organize and save your to do list.
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To Do
+
+> A simple minimalist browser based application to create save and manage your daily task.
+
+![screenshot](./app_screenshot.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- Html/Css
+- Javascript
+- Webpack
+
+## Live Demo
+
+[Live Demo Link](https://amazing-shaw-03b76e.netlify.app)
```

Comment:
### [Optional]

It would be nice to update your live demo link.

Reply:
- Live demo link is as it is supposed to be now. Well done! :+1: :clap: